CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG for Sulu
 * dev-develop
     * FEATURE     #702 [AdminBundle]    Added sortings to user settings and changed default url for activities
     * BUGFIX      #697 [ContactBundle]  Set VAT number field optional
+    * FEATURE     #697 [ContactBundle]  Refactored HTTP cache, introduced Varnish support. See 38af8da73c929f9f57bb87a8973a1ee55dccee29
```

Comment:
Should be `HttpCacheBundle/HttpCache`. The original code was in `CoreBundle` and the `HttpCache` component. It is mainly `HttpCacheBundle` though.